packages/ripple-nuxt-tide/modules/preview/templates/plugin.js

```diff
@@ -1,9 +1,18 @@
-export default function ({ $axios, app, redirect }) {
+import { logger } from '@dpc-sdp/ripple-nuxt-tide/lib/core'
+
+export default function ({ $auth, $axios }) {
+
+  $auth.onError((error, name, endpoint) => {
+    if (process.server) {
+      logger.debug('Auth error: "%O"', name, { error, label: 'Tide preview' })
+    }
+  })
 
   $axios.onError(error => {
-    // Redirect to login page again if both token and refresh token have expired.
-    if (error.name?.includes('ExpiredAuthSessionError')) {
-      redirect('/oauth/login')
```

Comment:
@tim-yao general question? If we are not redirecting, then what will happen after the expired auth session?

Reply:
It should handled by auth module [resetonerror](https://auth.nuxtjs.org/api/options/#resetonerror) I have it enabled https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-tide/modules/preview/module.js#L49

But that does not seem work for some case, that's why I have it in https://github.com/dpc-sdp/ripple/pull/888/files#diff-ae894b114c4ab959cac6d35451789a5ca617aa59996151f7e0bffa6392172336R100 to retry. So far looks it handle all the errors in our tests.